admin/security.md

```diff
@@ -11,10 +11,11 @@ are available under **Manage** > **Admin** > **Infrastructure**.
 Enabling **HTTP Strict Transport Security** sets [the
 `strict-transport-security`
 header](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Strict-Transport-Security)
-for Coder. If you're serving Coder over TLS, you can safely enable this header
-(most ingress options add this header). After you set this header, all users
-revisiting the site will use HTTPS, since clients may no longer access the site
-via HTTP.
+for Coder. If you are serving Coder over HTTPS, we recommend enabling HTTP
```

Comment:
I think we can shorten this without losing meaning, I'll add a commit to this branch